packages/augur-ui/src/modules/common/icons.tsx

```diff
@@ -308,6 +308,14 @@ export const DirectionArrow = (
   </svg>
 );
 
+export const SortByIcon = (
+  <svg viewBox="0 0 14 14" fill="none">
+    <path d="M4.78481 6.2001L4.51041 5.1545H2.69841L2.41601 6.2001H0.76001L2.58001 0.600098H4.59041L6.43041 6.2001H4.78481ZM4.19761 3.9193L3.95681 3.0001C3.90161 2.7961 3.83281 2.5329 3.75281 2.2105C3.67281 1.8881 3.62001 1.6561 3.59281 1.5161C3.57041 1.6457 3.52534 1.85903 3.45761 2.1561C3.38988 2.45316 3.24054 3.03983 3.00961 3.9161L4.19761 3.9193Z" fill="#D7DDE0"/>
+    <path d="M5.77756 13.4H1.41516V12.4648L3.83996 9.02485H1.47276V7.80005H5.71676V8.73125L3.29196 12.1784H5.77756V13.4Z" fill="#D7DDE0"/>
+    <path d="M10.8064 9.4001V0.600098H9.2064V9.4001H6.8064L10.0064 13.4001L13.2064 9.4001H10.8064Z" fill="#D7DDE0"/>
+  </svg>
+);
+
```

Comment:
one minor change, can we drop the `fill` hard colors on the attributes here and instead use a stylesheet to color the paths in this SVG? we do this in the off chance we want to change their colors else where on the site and it's more difficult to do if it's defined directly on the elements.

something like this:
```
> svg > path {
fill: @color-primary-text;
}
```

Reply:
Updated. Thank you.